content/en/docs/userguide/locality_loadbalance.md

```diff
@@ -288,12 +290,13 @@ $ Hello version: region.zone1.subzone2, instance: helloworld-region.zone1.subzon
 
 - Relabel the locality of the ambient-worker3 same as the worker2 and test. 
 ```
-kubectl label node ambient-worker3 topology.kubernetes.io/region=region
```

Comment:
why do you remove this region label

Reply:
Because the `region` label has already been added in the beginning: https://github.com/kmesh-net/website/blob/97b2dab89f7d8e47ae76e837f12d2b1ab548139c/content/en/docs/userguide/locality_loadbalance.md?plain=1#L83-L85

Here we want to change the locality of ambient-worker3, so we only need to adjust the `zone` and `subzone` labels.